### PR TITLE
CLOUD-435: Updated docs for Managed RonDB: Online Reconfiguration

### PR DESCRIPTION
**Pull request by mronstro**
Fixes https://hopsworks.atlassian.net/browse/CLOUD-435

### CODE REVIEW

**Review by smkniazi — APPROVED** [review]

**Review by maismail — CHANGES_REQUESTED** [review]

File: docs/setup_installation/common/rondb.md [hunk 1]
```diff
@@ -34,30 +71,54 @@ If you enable Managed RonDB you will see a basic configuration page where you ca
 
 First, you need to select the instance type and local storage size for the data nodes.
 These are the database nodes that will store data.
-RonDB is an in-memory database, so in order to fit more data you need to choose an instance type with more memory.
-Local storage is used for offline storage of recovery data, and requires a disk at least 40 GiB plus 1.8 times the memory size of the data node VM.
+
+RonDB in Hopsworks is an in-memory database, so in order to fit more data you need to choose an instance type with more memory.
+Local storage is used for offline storage of recovery data, and requires a disk of at least 40 GiB plus 1.8 times the memory size of the data node VM. RonDB does support on-disk columns that can be used for on-disk features, this will be accessible from Hopsworks in some future version.
```
[the remark this answers is not in the document]
Comment (mronstro) [reply docs/setup_installation/common/rondb.md:76]: Agree on removing it from docs.

File: docs/setup_installation/common/rondb.md [hunk 2]
```diff
@@ -91,12 +152,222 @@ You can choose the number of nodes, the instance type and local storage size.
...
+the MySQL Servers is low and one could use 2 x 4 VCPUs. In the morning business picks up and one can increase
+the use to 2 x 16 VCPUs. In the evening the business is heavier and one needs 2 x 32 VCPUs to meet the demand.
+Finally weekend rushes is handled with 3 x 32 VCPUs.
+
+## Why Reconfiguring Storage Layer (RonDB Data nodes)
+Reconfiguration of the RonDB Data nodes is a bit more rare since the database resides in memory. At the moment
+we only support increasing the size of the RonDB data nodes to ensure that we always have room for the database.
+Storing the database in-memory provides much lower latency to online applications.
+
+Thus reconfiguring of the Storage Layer is mainly intended to meet long-term needs of the business in terms of
```
[the remark this answers is not in the document]
Comment (mronstro) [reply docs/setup_installation/common/rondb.md:189]: ok, will add

File: docs/assets/images/setup_installation/managed/common/rondb/reconfig_console_deactivate.png
[the remark this answers is not in the document]
Comment (mronstro) [reply docs/assets/images/setup_installation/managed/common/rondb/reconfig_console_deactivate.png:1]: Agree, but the work required for the change isn't worth the effort. Instead I will take it into account when updating the docs for Reconciliation which should not be that far away, fairly soon after vacations.

File: docs/setup_installation/common/rondb.md [hunk 3]
```diff
@@ -3,9 +3,46 @@ For applications where Feature Store's performance and scalability is paramount
 Managed [RonDB](https://www.rondb.com/). You don't need to worry about configuration as [managed.hopsworks.ai](https://managed.hopsworks.ai) will
 automatically pick the best options for your setup.
 
+#Selecting Cloud
+
```
Comment (maismail) [inline docs/setup_installation/common/rondb.md:7]: no need for this header
```suggestion
```

**Review by maismail — CHANGES_REQUESTED** [review]

**Review by maismail — APPROVED** [review]

**Pull request merged**